run_kraken.py

```diff
@@ -352,7 +364,11 @@ def main(cfg):
 
             iteration += 1
             logging.info("")
-
+        # send telemetry
+        request_id=str(uuid.uuid1())
```

Comment:
Thoughts on using the Krkn generated uuid: https://github.com/redhat-chaos/krkn/blob/main/run_kraken.py#L150?